mm2src/mm2_main/src/rpc/dispatcher/dispatcher.rs

```diff
@@ -282,3 +270,40 @@ async fn gui_storage_dispatcher(
         _ => MmError::err(DispatcherError::NoSuchMethod),
     }
 }
+
+/// `lightning` dispatcher.
+///
+/// # Note
+///
+/// `lightning_method` is a method name with the `lightning::` prefix removed.
+#[cfg(not(target_arch = "wasm32"))]
+async fn lightning_dispatcher(
```

Comment:
Great, thank you for moving the lightning related RPCs outside of `dispatcher_v2`!

Reply:
It's much cleaner this way :)